test/commands/org/login/login.sfdx-url.nut.ts

```diff
@@ -56,4 +56,12 @@ describe('org:login:sfdx-url NUTs', () => {
     const output = getString(result, 'shellOutput.stdout');
     expect(output).to.include(`Successfully authorized ${username} with org ID`);
   });
+
+  it('should authorize an org using sfdx-url (human readable)', () => {
+    env.setString('TESTKIT_EXECUTABLE_PATH', `echo '${authUrl}' | ${env.getString('TESTKIT_EXECUTABLE_PATH')}`);
+    const command = 'org:login:sfdx-url -d --sfdx-url-stdin';
+    const result = execCmd(command, { ensureExitCode: 0 });
+    const output = getString(result, 'shellOutput.stdout');
+    expect(output).to.include(`Successfully authorized ${username} with org ID`);
+  });
```

Comment:
@mshanemc the `execCmd` function does not have a way to pipe from stdin. I tried to hack it using TESTKIT_EXECUTABLE_PATH but I'm not sure if it works because I did not understand how to run nut tests. Could you run it?

Reply:
I couldn't get it to work using `execCmd`... but I was able to get the piping working by directly using `shelljs`

```typescript
import * as exec from 'shelljs';


    const res = exec.cat(authUrl).exec('bin/dev org:login:sfdx-url -d --sfdx-url-stdin', { silent: true });
    expect(res.stdout).to.include(`Successfully authorized ${username} with org ID`);
```